application/world/services/auto_bible_generator.py

```
@@ -723,67 +740,21 @@ async def _save_worldbuilding(self, novel_id: str, worldbuilding_data: Dict[str,
             return {}
-        if bible is None:
+        try:
+            wb = self.worldbuilding_service.get_worldbuilding(novel_id)
+            return {
+                "core_rules": wb.core_rules,
+                "geography": wb.geography,
+                "society": wb.society,
+                "culture": wb.culture,
+                "daily_life": wb.daily_life
+            }
+        except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)